README.md

```diff
@@ -1,2 +1,54 @@
 # resource-auditing
-Audit changes to K8s resources (e.g. NetworkPolicies)
+
+## Overview
+Project resource-auditing provides an auditing system for network policy-related, Kubernetes and Antrea custom-defined resources that tracks creations, reconfigurations, and deletions of these resources, stored as YAML files backed by a Git repository. The system also comes with a CLI for querying and filtering the repository for changes based on filters like date range or service account responsible for those changes, as well as a tagging and rollback feature for reverting the cluster state if the current cluster state is undesirable. A webUI service is linked to the repository, allowing for easy visualization of the entire history of resource configurations.
```

Comment:
please wrap your lines at 80 lines, you can configure your text editor to do it automatically